File: packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerPackage.java
package expo.modules.imagepicker;

import android.content.Context;

import org.unimodules.core.BasePackage;
import org.unimodules.core.ExportedModule;

import java.util.Collections;
import java.util.List;

public class ImagePickerPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ImagePickerModule(context));
  }
}

File: android/versioned-abis/expoview-abi40_0_0/src/main/java/abi40_0_0/expo/modules/imagepicker/ImagePickerPackage.kt
package abi40_0_0.expo.modules.imagepicker

import android.content.Context
import abi40_0_0.org.unimodules.core.BasePackage

class ImagePickerPackage : BasePackage() {
  override fun createExportedModules(context: Context) = listOf(ImagePickerModule(context))
}
